README.md

```diff
@@ -2,284 +2,64 @@
 
 The Selected Tests service is used to predict which tests need to run based on code changes.
```

Comment:
Might be worth mentioning here that it depends on a running instance of evergreen and is intended to be used in tandem with evergreen.